Documentation/UsingSetting/PageTSconfig.rst

```diff
@@ -31,60 +31,47 @@ this is not recommended anymore.
 Setting the Page TSconfig globally
 ==================================
 
-Many page TSconfig settings can be set globally. This is useful for
-installations that contain only one site and use only one sitepackage extension.
+.. versionadded:: 12.0
+   Starting with TYPO3 12.0 page TSconfig in a file named
+   :file:`Configuration/page.tsconfig` in an extension is automatically
+   loaded during build time.
 
-Extensions supplying custom default Page TSconfig that should always be included,
-can also set the Page TSconfig globally.
+Global page TSconfig should be stored within an extension, usually a sitepackage
+extension. The content of the file :file:`Configuration/page.tsconfig` within
+and extension is automatically loaded during build time.
 
-Use extension API function :code:`addPageTSConfig()` in the
-:file:`ext_localconf.php` file of your extension:
+It is possible to load other TSconfig files with the import syntax within this
+file:
 
-.. code-block:: php
-   :caption: EXT:my_sitepackage/ext_localconf.php
+.. code-block:: tsconfig
+   :caption: EXT:my_sitepackage/Configuration/page.tsconfig
 
-   use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+   @import 'EXT:myexample/Configuration/TSconfig/Page/Basic.tsconfig
+   @import 'EXT:myexample/Configuration/TSconfig/Page/Mod/Wizards/NewContentElement.tsconfig'
 
-   ExtensionManagementUtility::addPageTSConfig('
-      TCEMAIN.table.pages {
-         disablePrependAtCopy = 1
-      }
-   ');
 
-There is a global `TYPO3_CONF_VARS` value called
-:ref:`$GLOBALS['TYPO3_CONF_VARS']['BE']['defaultPageTSconfig'] <t3coreapi:typo3ConfVars_be_defaultPageTSconfig>`.
+Many page TSconfig settings can be set globally. This is useful for
+installations that contain only one site and use only one sitepackage extension.
 
-The API function above adds content to that array. The array value itself
-however should **not** be changed or set directly (for example in the
-:file:`LocalConfiguration.php`).
+Extensions supplying custom default Page TSconfig that should always be included,
+can also set the Page TSconfig globally.
 
-It is best practice to use the above API method to add your default
-Page TSconfig in a project-specific
-:doc:`sitepackage <t3sitepackage:Index>` extension.
+Global page TSconfig, compatible with TYPO3 11 and 12
+-----------------------------------------------------
 
-Use the :typoscript:`@import '...'` syntax to keep the Page TSconfig in a
-separate file.
+In TYPO3 11 installations the content of file:`Configuration/page.tsconfig`
+is not loaded automatically yet. You can achive compatibility with both
+TYPO3 11 and 12 by importing the content of this file with the API function
```

Comment:
is it then imported twice in v12 // does this need a version condition surrounding it?